repo/packages/L/linkerd/5/marathon.json.mustache

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "linkerd",
+  "instances": {{linkerd.instances}},
+  "cpus": {{linkerd.cpus}},
+  "mem": {{linkerd.mem}},
+  "acceptedResourceRoles": [{{{linkerd.resource-roles}}}],
+  "constraints": [["hostname", "UNIQUE"]],
+  "maintainer": "hello@buoyant.io",
```

Comment:
It does work, I probably cribbed it from `marathon-lb`'s `marathon.json.mustache`. I'll remove it.